src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Innlogginga har feila.</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Innloggingsinformasjonen vart ikkje funnen.</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Authentication request could not be processed due to a system problem.</source>
+                <target>Innlogginga vart ikkje fullført på grunn av ein systemfeil.</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>Invalid credentials.</source>
+                <target>Ugyldig innloggingsinformasjon.</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>Cookie has already been used by someone else.</source>
+                <target>Informasjonskapselen er allereie brukt av ein annan brukar.</target>
+            </trans-unit>
+            <trans-unit id="6">
+                <source>Not privileged to request the resource.</source>
+                <target>Du har ikkje åtgang til å be om denne ressursen.</target>
+            </trans-unit>
+            <trans-unit id="7">
+                <source>Invalid CSRF token.</source>
+                <target>Ugyldig CSRF-teikn.</target>
+            </trans-unit>
+            <trans-unit id="8">
+                <source>Digest nonce has expired.</source>
+                <target>Digest nonce er ikkje lenger gyldig.</target>
+            </trans-unit>
+            <trans-unit id="9">
+                <source>No authentication provider found to support the authentication token.</source>
+                <target>Fann ingen innloggingstilbydar som støttar dette innloggingsteiknet.</target>
+            </trans-unit>
+            <trans-unit id="10">
+                <source>No session available, it either timed out or cookies are not enabled.</source>
+                <target>Ingen sesjon tilgjengeleg. Sesjonen er anten ikkje lenger gyldig, eller informasjonskapslar er ikke skrudd på i nettlesaren.</target>
+            </trans-unit>
+            <trans-unit id="11">
+                <source>No token could be found.</source>
+                <target>Fann ingen teikn.</target>
```

Comment:
In this case we are loosing a bit of context here, is this referring to `CSRF token`?

If so maybe use `CSRF-teikn` as `teikn` in nn i far more generic word.

Reply:
The message is used in context of "remember me" tokens.

Reply:
Ok, does the interface make this clear?

Reply:
These are authentication tokens. I have used "innloggingsteikn" in another place. Not sure if that is _good_, but at least it clarifies it.

Reply:
Ok with you @derrabus ?

Reply:
I don't speak Norwegian, so I cannot decide if a translation is good. 😉